templates/student_rent.html

```
@@ -182,14 +154,14 @@ <h2 class="h6 mb-0 d-flex align-items-center justify-content-between">
         <div class="alert alert-info mb-0">
             <strong>Not Yet Due</strong>
-            <br>Rent will be due starting {{ payment_due_date.strftime('%B %d, %Y') }}
+            <br>Rent will be due starting {% if status.due_date %}{{ status.due_date.strftime('%B %d, %Y') }}{% else %}soon{% endif %}
         </div>
         {% else %}
         {% if status.is_paid and status.is_waived %}
         <div class="alert alert-info mb-0">
             <strong>This rent period has been waived.</strong>
-            {% if active_waivers and active_waivers|length > 0 %}
-                {% set current_waiver = active_waivers[0] %}
+            {% if view.active_waivers and view.active_waivers|length > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)